CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Minor: Added context menu action to toggle visibility of offline tabs. (#5318)
 - Minor: Report sub duration for more multi-month gift cases. (#5319)
 - Minor: Improved error reporting for the automatic streamer mode detection on Linux and macOS. (#5321)
+- Minor: Remove unnecessary redirect of player.twitch.tv URLs (#5327)
```

Comment:
I added a missing `.`, changed the PR number, made the category a `Dev` entry since it doesn't actually change anything, and updated the wording to better reflect what it does (which we can be more free with since it'd be a dev entry). You'd also need to move it to the bottom of the changelog entries since it's now a dev entry, which is why I haven't used the suggestion functionality.

`- Dev: Changed the order of the query parameters for Twitch player URLs. (#5326)`